frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -1,37 +1,79 @@
 import React from "react";
+import styled from "@emotion/styled";
 import {
   Avatar as MuiAvatar,
+  Box,
   ClickAwayListener,
-  Grid,
-  Grow,
+  Grow as MuiGrow,
   IconButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem as MuiMenuItem,
   MenuList,
-  Paper,
+  Paper as MuiPaper,
   Popper,
   Typography,
 } from "@material-ui/core";
 import Cookies from "js-cookie";
 import jwtDecode from "jwt-decode";
-import styled from "styled-components";
-
-const UserPhoto = styled(IconButton)`
-  ${({ theme }) => `
-  padding: 1px 0 ${theme.spacing(0.5)}px 12px;
-  margin-right: ${theme.spacing(0.5)}px;
-  `}
-`;
-
-const Avatar = styled(MuiAvatar)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.text.secondary}
-  `}
-`;
-
-const Initials = styled(Typography)`
-  ${({ theme }) => `
-  color: ${theme.palette.accent.main}
-  `}
-`;
+
+const UserPhoto = styled(IconButton)({
+  padding: "12px",
+  "&:hover": {
+    background: "#2d3db4",
+  },
+  "&:active": {
+    background: "#2938a5",
+  },
+});
+
+const Avatar = styled(MuiAvatar)({
+  backgroundColor: "#dce7f4",
+  height: "28px",
+  width: "28px",
+});
+
+const AvatarBackdrop = styled(MuiAvatar)({
+  backgroundColor: "#f6faff",
+  height: "32px",
+  width: "32px",
+});
+
+const Initials = styled(Typography)({
+  color: "#0d1030",
+  opacity: 0.6,
+  fontSize: "14px",
+  fontWeight: 500,
+});
+
+const Paper = styled(MuiPaper)({
+  width: "266px",
+  border: "1px solid #e2e2e6",
+  boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+});
+
+const UserProfileMenuItem = styled(MuiMenuItem)({
+  "&:focus": {
+    background: "transparent",
+  },
+  "&:hover": {
+    background: "transparent",
```

Comment:
I thought we were going to use the hover/active states from the select for this as well.

Reply:
i thought we were adding hover states to the list items that follow, but not to the user profile section (like the github profile menu). but i'll follow up to add an interface to the user component so we can render out more menu items and test how it all looks together. will leave a to-do